redirects-list.json

```diff
@@ -4,5 +4,6 @@
   "^/(?:docs|documentation)(/.*)?$            /doc$1",
   "^/(.+)/$                                   /$1",
 
-  "^/doc/user-guide/remote-projects$          /doc/user-guide/remote-objects 302"
+  "^/doc/user-guide/remote-projects$          /doc/user-guide/remote-objects 302",
+  "^/doc/get-started/(.+)                     /doc/get-started 302"
```

Comment:
yep, I used that file as a source of examples. What's the difference though?

Reply:
> What's the difference

In this case I think that there's no difference between `/(.+)` and `(/.*)?` since matches are not used later (`$1`, etc.) but not sure whether the `$` could make a difference... I like to add it to be more strict as a best practice. In general, `(/.*)?$` is tested and proven, but this is minor. Up to you/ @iterative/websites

Reply:
> making /doc/get-started redirect to /doc/get-started in an infinite loop

That's not happening in dvc.org atm but I do see your point. Should we also update the dvc.org and cml.dev repos with the simpler `/(.+)` ? Should we add `$` at the end too? Thanks

Reply:
> if we decided that the PR is ok

Sorry, not trying to block this @aguschin . It's pending a review by @iterative/websites I think. My only pending Q is whether the regex needs `$` at the end, really. I'll approve it for now.

Reply:
I don't think the $ at the end is required since the default match behavior is greedy, though I think the regex would also work with it and technically be more specific. Just a preference thing.

Reply:
OK let's add it for consistency and do it's extra strict, I say.

```suggestion
  "^/doc/get-started/(.+)$                    /doc/get-started 302"
```

Reply:
I'll try it out for consistency's sake and see how it goes on the deploy.